src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,149 @@
+			"targetWarning": "Le lien s'ouvrira dans une nouvelle fenêtre de navigateur",
+			"yesBtn": "Oui",
+			"cancelBtn": "Annuler"
+
+		}
+	},
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			settings,
+			queryString = window.location.search,
+			urlParams = new URLSearchParams( queryString ),
+			originalURL = urlParams.get( "exturl" ),
+			$elm;
+		if ( elm ) {
+			$elm = $( elm );
+			settings = $.extend(
+				true,
+				window[ componentName ],
+				wb.getData( $elm, componentName )
+
+			);
+
+			$elm.data( componentName, settings );
+
+			if ( settings.url ) {
+				$( this ).attr( "href", settings.url + "?exturl=" +  encodeURIComponent( this.href ) );
+			}
+
+			if ( !i18n ) {
+				i18n = $.extend( true, i18nDict[ wb.lang || "en" ], settings.i18n || {} );
+			}
+
+			// This conditional statement for a middle static exit page, to retrieve the URL to the non-secure site.
+			if ( $elm.hasClass( exiturlparam ) ) {
+				this.outerHTML = "<a href='" + originalURL + "'>" + originalURL + "</a>";
+			}
+			wb.ready( $elm, componentName );
+		}
+	};
+
+$document.on( "click", selector, function( event ) {
+
+	var elm = event.currentTarget,
+		$elm = $( elm ),
+		wrapper,
+		targetAttribute = "",
+		moDal = document.createDocumentFragment(),
+		tpl = document.createElement( "div" ),
+		settings =  $elm.data( componentName ),
+		msgboxHeader = i18n.msgboxHeader,
+		yesBtn = i18n.yesBtn,
+		cancelBtn = i18n.cancelBtn,
+		exitMsg = i18n.exitMsg;
+
+	if ( settings.i18n ) {
+		msgboxHeader =  settings.i18n.msgboxHeader || i18n.msgboxHeader;
+		yesBtn = settings.i18n.yesBtn || i18n.yesBtn;
+		cancelBtn = settings.i18n.cancelBtn || i18n.cancelBtn;
+		exitMsg = settings.i18n.exitMsg || i18n.exitMsg;
+	}
+
+	if ( !settings.url ) {
+
+		event.preventDefault();
+	}
+
+	if ( this.getAttribute( "target" ) !== null ) {
+		targetAttribute = "target='" + this.getAttribute( "target" ) + "'";
+		exitMsg = exitMsg  + " " + i18n.targetWarning;
+	}
+
+	if ( document.getElementById( moDalId ) ) {
+		var tplModal = document.getElementById( moDalId );
+		tplModal.remove();
+	}
+
+	if ( !settings.url ) {
+		tpl.innerHTML = "<section id='" + moDalId + "' " + "class='mfp-hide modal-dialog modal-content overlay-def'>" +
+			"<header class='modal-header'><h2 class='modal-title'>" + msgboxHeader + "</h2></header>" +
+			"<div class='modal-body'>" +
+			"<p>" + exitMsg + "</p>" +
+			"</div>" +
+			"<div class='modal-footer'>" +
+			"<ul class='list-inline text-center'>" +
+			"<li><button class='btn btn-primary popup-modal-dismiss pull-left' id='eCancel'>" + cancelBtn + "</button></li>" +
+			"<li><a class='btn btn-default pull-right popup-modal-close'" + targetAttribute + " href='" + this.getAttribute( "href" ) + "' type='button'>" + yesBtn + "</a></li>" +
```

Comment:
We have discussed about this pattern and PP and WAWG agree that

Reverse the order and change the style.
```
<ul class="list-inline text-center">
<li><a href="http://csszengarden.com/219" class="btn btn-primary pull-right popup-modal-close">Yes</a></li>
<li><button class="btn btn-default popup-modal-dismiss pull-left">Cancel</button></li>
</ul>
```

"Yes" option should be primary, it's the natural order and common pattern order for modal like this. Also, the user has already click on a link to navigate away, so it is high risk they will navigate away anyway.

Remove the ID of the button `id="eCancel"`

Reply:
done